python_modules/dagster/dagster/_core/execution/plan/execute_step.py

```diff
@@ -427,19 +445,24 @@ def _type_check_and_store_output(
         yield evt
 
 
+def _get_output_asset_info(
```

Comment:
This seems like it would make more sense as a method on `OutputContext`. If we don't want to expose `AssetOutputInfo` to users, we could avoid marking it `@public`.

Reply:
makes sense! turns out there was an existing method on `OutputContext` that already returned this